packages/docusaurus-theme-classic/src/theme/MDXComponents/index.tsx

```diff
@@ -12,8 +12,8 @@ import MDXA from '@theme/MDXComponents/A';
 import MDXPre from '@theme/MDXComponents/Pre';
 import MDXDetails from '@theme/MDXComponents/Details';
 import MDXHeading from '@theme/MDXComponents/Heading';
-import MDXUl from '@theme/MDXComponents/Ul';
```

Comment:
why remove MDXUl? 

It is a "safe" component meant to be swizzled and customized. Removing it means a breaking change.